packages/react-composites/tests/browser/chat/ChatComposite.test.ts

```diff
@@ -17,6 +17,13 @@ const stubTimestamps = (page: Page): void => {
   }, messageTimestampId);
 };
 
+const compositeLoaded = async (page: Page): Promise<void> => {
+  await page.waitForLoadState('load');
+  await page.waitForSelector(dataUiId(IDS.sendboxTextfield));
+  await page.waitForSelector(dataUiId(IDS.participantList));
+  await page.waitForTimeout(1000);
```

Comment:
Does this add a second to each test? Why is this necessary?

This can also introduce flakiness to the test as 1 second may not be enough for things to settle on a loaded CI server.

Reply:
Not to each test. Adds 1 second per browser page to overall test run.
So in this case, a total of 2 seconds.
And yes, it is a flaky test but I couldn't find a better way. 
Improving this is in my todo.

Reply:
Total time increased is 1 second per browser page.
So a total of 2 seconds are added to these tests.
Without doing this, messages sent at the beginning were not getting delivered to the recipient.
Couldn't find a better way to handle this. 
Improving this is in my todo list.

Reply:
As discussed IRL, let's add a comment explaining why we have this wait.

I am confident this is discovering a real bug in our implementation. We lose messages that appear right when the stateful client is being initialized.

Reply:
The chat composite in storybook sometimes loses the first bot message, I assumed it was a strange storybook rendering issue where it rerenders somepoint after the first message but could be the same underlying bug

Reply:
Looks like it.